submitit/local/test_local.py

```diff
@@ -78,6 +79,15 @@ def failing_job() -> None:
     assert "Failed on purpose" in traceback
 
 
+def test_pickle_output_from_main(tmp_path: Path) -> None:
+    class MyClass:
+        pass
+
+    executor = local.LocalExecutor(tmp_path)
+    job = executor.submit(MyClass.__call__)
```

Comment:
With standard pickle, this would fail when pickling the output:
```
pickle.dump(obj, ofile, pickle.HIGHEST_PROTOCOL)
Can't pickle <class 'submitit.local.test_local.MyClass'>: attribute lookup MyClass on submitit.local.test_local failed
```

Reply:
Thanks for the minimalist repro :-)